nimble/host/mesh/src/prov.c

```diff
@@ -727,6 +727,14 @@ static void prov_start(const u8_t *data)
 		return;
 	}
 
+	/* Public Key value is hardcoded in prov_invite() */
+	if (data[1] != 0x00) {
```

Comment:
so maybe we should have define for this used here and in prov_invite()

Reply:
Done

Reply:
Done